truffle/com.oracle.truffle.dsl.processor/src/com/oracle/truffle/dsl/processor/InstrumentRegistrationProcessor.java

```diff
@@ -57,24 +57,34 @@ public SourceVersion getSupportedSourceVersion() {
         return SourceVersion.latest();
     }
 
+    private static final int NumPropertiesPerEntry = 4;
```

Comment:
Yeah, I had that before, and thought is was a style violation because checkstyle complained. But the `final` was missing... will fix.